annotation_full.wdl

```diff
@@ -222,10 +240,11 @@ task stage {
    String input_file
 
    command <<<
-       set -e
+       set -euo pipefail
        if [ $( echo ${input_file}|egrep -c "https*:") -gt 0 ] ; then
            wget ${input_file} -O ${target}
        else
+           set +o pipefail
```

Comment:
you shouldn't need this second pipefail command since it is already on line 243

Reply:
@scanon added this one, i don't quite remember why, but something about the following line could fail if not?